protos/core.proto

```diff
@@ -152,6 +152,8 @@ message Table {
   // Pre/post operations.
```

Comment:
github won't let me comment on the relevant line, but please change `string where = 8;` to `string where = 8 [deprecated = true];`

Reply:
Ah very cool